src/validator/MaintainValidator.cpp

```diff
@@ -51,7 +51,7 @@ Status CreateTagValidator::validateImpl() {
 
 Status CreateTagValidator::toPlan() {
     auto *plan = qctx_->plan();
-    auto doNode = CreateTag::make(qctx_,
+    auto doNode = CreateTagNode::make(qctx_,
```

Comment:
Is `genSingleNodePlan` better？

Reply:
maybe they are the same or I will unify them later